WeakAuras/RegionTypes/RegionPrototype.lua

```diff
@@ -525,7 +540,9 @@ function WeakAuras.regionPrototype.AddExpandFunction(data, region, cloneId, pare
       end
       region.justCreated = nil;
       region:SetFrameLevel(WeakAuras.GetFrameLevelFor(region.id));
-
+      if WeakAuras.IsOptionsOpen() then
```

Comment:
I would prefer if expand doesn't need to check if the options are open. Was the are any problem with the old code for setting the script?

Reply:
If my memory doesn't fail me, the clicking was working if you clicked on the green eye icon to show regions, but not if you clicked the buttons, i'll move the code back to the DisplayButton properly

Reply:
yeah indeed, after reverting the change, any modification made on an aura which call WeakAuras.Add break ClickToPick

Reply:
fixed with https://github.com/WeakAuras/WeakAuras2/pull/1270/commits/e2d3897b6c892758eb05ecd33b963f20f5c63c29